lib/rubocop/cop/rspec/excessive_docstring_spacing.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for excessive whitespace in example descriptions.
```

Comment:
Do you mean `Checks for excessive whitespace in examples and example groups`?

I think that "example descriptions" is better here, because that is what's actually being checked - whereas examples & example groups are the whole section of code: description + body; but this cop doesn't check the body for excessive whitespace.

However, I've not worked within the RSpec community much so am still learning the names, so happy to go with your preference if you disagree :)